closure/compiler/closure_js_binary.bzl

```diff
@@ -67,12 +68,20 @@ def _impl(ctx):
     args += ["--debug"]
   elif is_using_closure_library(srcs):
     args += ["--define=goog.DEBUG=false"]
+  if is_using_closure_library(srcs) and language_out in ["ECMASCRIPT6_TYPED",
+                                                         "ECMASCRIPT6_STRICT",
+                                                         "ECMASCRIPT5_STRICT"]:
+    args += ["--define=goog.STRICT_MODE_COMPATIBLE"]
+  if contains_file(srcs, ctx.attr._soyutils_usegoog):
```

Comment:
Another alternative if you prefer, would possibly be to just write a one-off method that says: `for src in srcs: if src.startswith("external/soyutils_usegoog/") and src.endswith("soyutils_usegoog.js"): ...`